packages/fractal/views/cli/new/docs/index.hbs

```diff
@@ -2,4 +2,4 @@
 title: {{ projectTitle }}
```

Comment:
this file itself should also be renamed to md extension

Reply:
Done